crates/eips/src/eip4895.rs

```diff
@@ -1,27 +1,17 @@
-//! Withdrawal type and serde helpers.
+//! [EIP-4895] Withdrawal type and serde helpers.
+//!
+//! [EIP-2930]: https://eips.ethereum.org/EIPS/eip-4895
 
 use alloy_primitives::{Address, U256};
 use alloy_rlp::{RlpDecodable, RlpEncodable};
 use alloy_serde::u64_hex;
-use serde::{Deserialize, Serialize};
 
 /// Multiplier for converting gwei to wei.
 pub const GWEI_TO_WEI: u64 = 1_000_000_000;
 
 /// Withdrawal represents a validator withdrawal from the consensus layer.
-#[derive(
-    Debug,
-    Copy,
-    Clone,
-    PartialEq,
-    Eq,
-    Default,
-    Hash,
-    RlpEncodable,
-    RlpDecodable,
-    Serialize,
-    Deserialize,
-)]
+#[derive(Debug, Copy, Clone, PartialEq, Eq, Default, Hash, RlpEncodable, RlpDecodable)]
+#[cfg_attr(feature = "serde", derive(serde::Serialize, serde::Deserialize))]
 #[cfg_attr(feature = "ssz", derive(ssz_derive::Encode, ssz_derive::Decode))]
```

Comment:
this feature currently does not exist in this crate, so we need to add this and the required deps, see rpc-types for reference
then we have to enable this in rpc-types if the ssz feature is enabled there